plugin-server/src/utils/db/db.ts

```diff
@@ -673,109 +654,70 @@ export class DB {
         return undefined
     }
 
-    // Exported for tests only
-    public async getGroupDataCache(teamId: number, groupIdentifier: GroupIdentifier): Promise<GroupCacheData> {
-        const data = await this.redisGet<{ properties: Properties; created_at: string } | null | 'missing'>(
-            this.getGroupDataCacheKey(teamId, groupIdentifier.index, groupIdentifier.key),
-            'missing'
-        )
-
-        if (data === 'missing' || !data) {
-            return { identifier: groupIdentifier, data: null, cached: !data }
-        }
-        return {
-            identifier: groupIdentifier,
-            data: { properties: data.properties, created_at: DateTime.fromISO(data.created_at).toUTC() },
-            cached: true,
-        }
+    public async updateGroupCache(
```

Comment:
Here's a cleaner look at the new function:

https://github.com/PostHog/posthog/blob/71649cbea619b1de13cb2e3fa993e73c26228b23/plugin-server/src/utils/db/db.ts#L657